docs/getting-started-guides/windows/README.md

```diff
@@ -0,0 +1,159 @@
+ip route add 192.168.2.0/24 via <IP of Win02 host>
+```
+
+**Win01**
+```
+docker network create -d transparent --gateway 192.168.1.1 --subnet 192.168.1.0/24 <network name>
+# A bridge is created with Adapter name "vEthernet (HNSTransparent)". Set its IP address to transparent network gateway
+netsh interface ipv4 set address "vEthernet (HNSTransparent)" addr=192.168.1.1
+route add 192.168.0.0 mask 255.255.255.0 192.168.0.1 if <Interface Id of the Routable Ethernet Adapter> -p
+route add 192.168.2.0 mask 255.255.255.0 192.168.2.1 if <Interface Id of the Routable Ethernet Adapter> -p
+```
+
+**Win02**
+```
+docker network create -d transparent --gateway 192.168.2.1 --subnet 192.168.2.0/24 <network name>
+# A bridge is created with Adapter name "vEthernet (HNSTransparent)". Set its IP address to transparent network gateway
+netsh interface ipv4 set address "vEthernet (HNSTransparent)" addr=192.168.2.1
+route add 192.168.0.0 mask 255.255.255.0 192.168.0.1 if <Interface Id of the Routable Ethernet Adapter> -p
+route add 192.168.1.0 mask 255.255.255.0 192.168.1.1 if <Interface Id of the Routable Ethernet Adapter> -p
+```
+
+## Starting the Cluster
+For now, the Kubernetes control plane continues to run on Linux and as a result a Windows only Kubernetes Cluster is not possible. 
+## Linux
+Use your preferred method to start Kubernetes cluster on Linux. Please note that Cluster CIDR might need to be updated.
+## Windows
+### kubelet
+Run the following in a PowerShell window. Be aware that if the node reboots or the process exits, you will have to rerun the commands below to restart the kubelet
+
+1. Set environment variable *CONTAINER_NETWORK* value to the docker container network to use
+`$env:CONTAINER_NETWORK = "<docker network>"`
+
+2. Run *kubelet* executable using the below command
+`kubelet.exe --hostname-override=<ip address/hostname of the windows node>  --pod-infra-container-image="apprenda/pause" --resolv-conf="" --api_servers=<api server location>`
+
+### kube-proxy
+
+Run the following in a PowerShell window with administrative privileges. Be aware that if the node reboots or the process exits, you will have to rerun the commands below to restart the kube-proxy.
+
+1. Set environment variable *INTERFACE_TO_ADD_SERVICE_IP* value to a node only network interface. The interface created when docker is installed should work
+`$env:INTERFACE_TO_ADD_SERVICE_IP = "vEthernet (HNS Internal NIC)"`
+
+2. Run *kube-proxy* executable using the below command
+`.\proxy.exe --v=3 --proxy-mode=userspace --hostname-override=<ip address/hostname of the windows node> --master=<api server location> --bind-address=<ip address of the windows node>`
+
+## Scheduling PODs on Windows
+As Kubernetes control plane currently runs on Linux, the resulting cluster will have both Linux and Windows nodes. TO schedule PODs on Windows, `nodeSelector` constraint has to be set with label `beta.kubernetes.io/os` having value `windows` as shown in the example below,
```

Comment:
Suggested rephrase for clarity: "Because your cluster has both Linux and Windows nodes, you must explictly set the `nodeSelector` constraint to be able to schedule Pods to Windows nodes. You must set `nodeSelector` with the label `beta.kubernetes.io/os` to the value `windows`; see the following example: